### DISCUSSION TITLE
No 'Access-Control-Allow-Origin' header is present on the requested resource.

### DISCUSSION THREAD

**Discussion by gaurikalp** [EnsembleUI/ensemble#141] [General]
There is a cross-origin access error when we run the command: flutter run on Android Studio, as shown in the following image.

![image](https://user-images.githubusercontent.com/115991454/208658055-1aef5ff7-dbea-43ef-8484-6086172caad4.png)

**Comment by amin-nas** [comment]
This needs to be addressed on the server where you're hosting these images to either allow any domain, or specific domains such as `localhost` and `studio.ensemble.com`
